src/app/components/player/ExtraOptions.tsx

```diff
@@ -92,6 +93,26 @@ const ExtraOptions: React.FC = observer(() => {
             </>
           )}
         />
+        <Toggle
+          checked={player.buffs.thrallSpell}
+          setChecked={(c) => store.updatePlayer({ buffs: { thrallSpell: c } })}
+          label={(
+            <>
+              <img src={thrall.src} width={18} className="inline-block" alt="Thralls" />
+              {' '}
+              Use thralls
+              {' '}
+              <span
+                className="align-super underline decoration-dotted cursor-help text-xs text-gray-300"
+                data-tooltip-id="tooltip"
+                data-tooltip-content="Include thrall damage."
+              >
+                ?
+              </span>
```

Comment:
Should this just be Toggle's help attr?

Reply:
It seems like keeping it in the label is better if you want to keep the toggle component general purpose. You may want to remove the help completely since it's not used anywhere